docs/develop/terrain/using-terrain-testnet.mdx

```diff
@@ -155,4 +154,4 @@ npm start
 
 ## Advanced usage
 
-For more advanced use cases such as deploying to the testnet or mainnet, see [Terrain's readme](https://github.com/terra-money/terrain#readme).
+For more advanced use cases such as deploying to testnet or mainnet, please read [Terrain's readme](https://github.com/terra-money/terrain#readme).
```

Comment:
I would take out "please." 

"Please" and "thank you" aren't necessary in technical documentation unless you are asking the user to do a favor.